drivers/amazonec2/amazonec2.go

```diff
@@ -519,29 +521,6 @@ func (d *Driver) sshKeyPath() string {
 	return path.Join(d.storePath, "id_rsa")
 }
 
-func (d *Driver) updateDriver() error {
```

Comment:
The IP address on EC2 instances normally changes on stop/start, I guess this function was for that use case.